### PR TITLE
MCR-1814 show initial view page defined in pdf

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-1814).

### CODE REVIEW

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 8
           

Complexity increasing per file
==============================
- mycore-viewer/src/main/typescript/modules/base/components/MyCoReImageScrollComponent.ts  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493&bid=17310635)

**Review by kkrebs — APPROVED** [review]

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 1]
```diff
@@ -66,11 +67,11 @@ namespace mycore.viewer.components {
             if (this._settings.doctype == "pdf") {
                 this._pdfUrl = ViewerFormatString(this._settings.pdfProviderURL, { filePath: this._settings.filePath, derivate: this._settings.derivate });
                 var workerURL = this._settings.pdfWorkerURL;
-                PDFJS.workerSrc = workerURL;
+                pdfjsLib.workerSrc = workerURL;
                 var that = this;
                 var pdfLocation = this._pdfUrl;
-                PDFJS.getDocument(pdfLocation).then((pdfDoc) => {
-                    this._pdfDocument = <PDFDocumentProxy>pdfDoc;
+                (<any> pdfjsLib.getDocument(pdfLocation)).promise.then((pdfDoc:PDFDocumentProxy) => {
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:73]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 6
           

Complexity increasing per file
==============================
- mycore-viewer/src/main/typescript/modules/base/components/MyCoReImageScrollComponent.ts  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493&bid=17310635)

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 2]
```diff
@@ -29,9 +29,10 @@ namespace mycore.viewer.components {
             var that = this;
 
             // see MV-53
-            (<any>PDFJS).disableAutoFetch=true;
-            (<any> PDFJS).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
-            (<any> PDFJS).cMapPacked = true;
+            (<any>pdfjsLib).disableAutoFetch=true;
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:32]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 3]
```diff
@@ -29,9 +29,10 @@ namespace mycore.viewer.components {
...
+            (<any>pdfjsLib).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
+            (<any>pdfjsLib).cMapPacked = true;
+            (<any>pdfjsLib).disableOpenActionDestination = false;
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:35]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 4]
```diff
@@ -66,11 +67,11 @@ namespace mycore.viewer.components {
             if (this._settings.doctype == "pdf") {
                 this._pdfUrl = ViewerFormatString(this._settings.pdfProviderURL, { filePath: this._settings.filePath, derivate: this._settings.derivate });
                 var workerURL = this._settings.pdfWorkerURL;
-                PDFJS.workerSrc = workerURL;
+                pdfjsLib.workerSrc = workerURL;
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:70]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 5]
```diff
@@ -29,9 +29,10 @@ namespace mycore.viewer.components {
             var that = this;
 
             // see MV-53
-            (<any>PDFJS).disableAutoFetch=true;
-            (<any> PDFJS).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
-            (<any> PDFJS).cMapPacked = true;
+            (<any>pdfjsLib).disableAutoFetch=true;
+            (<any>pdfjsLib).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:33]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

File: mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts [hunk 6]
```diff
@@ -29,9 +29,10 @@ namespace mycore.viewer.components {
...
+            (<any>pdfjsLib).cMapPacked = true;
```
Comment (adlerre) [inline mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts:34]: Codacy Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on d5680b6c6143440523e0bab35d4b06900dfe8853! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5443-pull_request-947

**Review by kkrebs — APPROVED** [review]

**Pull request merged**